src/cmake/cmakeExecutable.ts

```diff
@@ -29,10 +29,12 @@ export async function getCMakeExecutableInformation(path: string): Promise<CMake
         const normalizedPath = util.platformNormalizePath(path);
         if (cmakeInfo.has(normalizedPath)) {
             const cmakeExe: CMakeExecutable = cmakeInfo.get(normalizedPath)!;
-            await setCMakeDebuggerAvailableContext(
-                cmakeExe.isDebuggerSupported?.valueOf() ?? false
-            );
-            return cmakeExe;
+            if (cmakeExe.isPresent) {
```

Comment:
Should we log an error if cmake is not present here? 

Is this the case where the user has installed cmake between the time when they opened vscode and their first invocation of configure?

Reply:
@elizabethmorrow Correct, this is the case, but actually broader. It's any case where they open vscode, then install, then do any action where we invoke cmake.exe. 

I would agree it's a good idea to log here, @qarni we should do a quick PR for this as well